src/assets/usage-examples-code/sprites/sprite_set_y/sprite_set_y-9-sprite-y.txt

```diff
@@ -0,0 +1,3 @@
+### sprite_set_y() usage example
```

Comment:
Improve this heading to follow the style used in existing usage examples.

Reply:
I have changed the heading